bundle/deploy/files/sync.go

```diff
@@ -14,12 +14,21 @@ func getSync(ctx context.Context, b *bundle.Bundle) (*sync.Sync, error) {
 		return nil, fmt.Errorf("cannot get bundle cache directory: %w", err)
 	}
 
+	internalDir, err := b.InternalDir()
+	if err != nil {
+		return nil, fmt.Errorf("cannot get bundle internal directory: %w", err)
+	}
+
 	opts := sync.SyncOptions{
-		LocalPath:   b.Config.Path,
-		RemotePath:  b.Config.Workspace.FilesPath,
+		LocalPath:  b.Config.Path,
+		RemotePath: b.Config.Workspace.FilesPath,
+		Include:    b.Config.Sync.Include,
+		Exclude:    b.Config.Sync.Exclude,
+
 		Full:        false,
 		CurrentUser: b.Config.Workspace.CurrentUser.User,
 
+		InternalDir:      internalDir,
```

Comment:
Why is `InternalDir` treated specially? I would expect it to be added to the set of include patterns.